src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.CodeFixes/CodeFixResources.resx

```diff
@@ -147,6 +147,9 @@
   <data name="ResultTypeTitle" xml:space="preserve">
     <value>Replace with ASP.NET Core result type</value>
   </data>
+  <data name="TypeUpgradeTitle" xml:space="preserve">
+    <value>Replace type with .NET Core type</value>
```

Comment:
Is it necessarily a .NET Core type? Or is it just a type replacement?

Reply:
No, I suppose not. I'll update this to make it more general.